IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,212 @@
+    <TaskItem Name="Store location size">Store the users input as "locationType".</TaskItem>
+
+    <Text>
+        It might also be helpful at this point to capture just how many people are in this town or area! The population of a particular
+        location in a game might increase or decrease over the cyourse of the game but for now you are just going to get a starting population.
+    </Text>
+
+    <TaskItem Name="Ask for population">Create a prompt to the user for the location population.</TaskItem>
+    <TaskItem Name="Store location population">Store the location population as an <b>integer</b></TaskItem>
+
+    <Text>
+        You are well on your way to make your first location in a game! What you need now is to build the area or "geography" around your
+        city. When you think of the location what kind of natural reasources surround it? Is it a small village in the center of a forest?
+        A colony in space surrounded by asteroid fields? An underground civilization built within a mine filled with enormous crystals?
+    </Text>
+    <Text>
+        You may also want to know what natural resources surround the location! Is is close to the ocean? Are there a lot of animals in the area
+        for hunting? Perhaps it is in a metropolis surrounded by nothing but desert?
+    </Text>
+
+    <TaskItem Name="Ask for geography">Create a prompt to the user for the geography.</TaskItem>
+    <TaskItem Name="Store geography">Store the geography of the location as "geography".</TaskItem>
+    <TaskItem Name="Ask for geography">Create a prompt to the user for the natural resyources around the location.</TaskItem>
+    <TaskItem Name="Store natural resources">Store the natural resyources as "naturalResyources".</TaskItem>
+
+    <Section Title="Displaying the information">
+    <Text>
+        Wow! You've got a lot of information going now! I think it is about time to start looking at how to display the information
+        in a way that looks good for the user. While you could just display the information out as "information: ", "user input" format,
+        it might be a little hard to read the information. Especially if you want to add more information.
+    </Text>
+    <Text>
+        You could consider changing the color of the information in each row to make it a little easier to read! Or perhaps
+        changing the format for the information.
+    </Text>
+
+    <TaskItem Name="Display Information">Display the information out to the user.</TaskItem>
+    <TaskItem Name="Alternate Colors">Alternate the colors displayed on each line to the user to make it easier to read.</TaskItem>
+
+    </Section>
+    <Section Title="Wrapping Up">
+    <Text>
+        By the end of this project your Display out should look something like this!
+    </Text>
+    <CodeBlock ReplIt="NobodyOdyssues/SettingBuilder"> City Builder</CodeBlock>
+    </Section>
+    <Section Title="Wrapping Up">
```

Comment:
Rename section to "What's Next?" and include a link to at least one other section that is semi-related or would be a good place to go after this.